src/Symfony/Component/Routing/Loader/DependencyInjection/ServiceRouterLoaderInterface.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Routing\Loader\DependencyInjection;
+
+/**
+ * Marker interface for router loader services.
+ */
+interface ServiceRouterLoaderInterface
```

Comment:
why the `Service` prefix?
wouldn't `RouterLoaderInterface` be a better name?

Reply:
`RouteLoaderInterface` according to https://github.com/symfony/symfony/pull/30926#discussion_r281145005

Reply:
The `Service` prefix was to stay consistent with the existing `ServiceRouterLoader` class.

Reply:
ok, so this confirms the prefix should be removed: "service" is an implementation detail that the interface shouldn't account for. And "Container" is the best practice now.
I agree with @chalasr's comments.

Reply:
Yes I have talked with @chalasr and I will deprecate / rename everyting directly in this PR 😄